### PR TITLE
enh(release): Add Centreon MBI 20.10.3 RN

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: fr/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -173,6 +173,19 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
...
+### 20.10.3
+
+`3 juin 2021`
+
+#### Correctifs
+
+- Correction des erreur JQuery avec MBI web client
+- Correction de l'exécution immédiate des jobs qui était effectuée dans le futur
+
+#### Enhancements
```
Comment (kduret) [inline fr/releases/centreon-commercial-extensions.md:185]:
```suggestion
#### Améliorations
```

**Review by kduret — APPROVED** [review]

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -173,6 +173,19 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon MBI
 
+### 20.10.3
+
+`3 juin 2021`
+
+#### Correctifs
+
+- Correction des erreur JQuery avec MBI web client
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:182]: des erreurs

File: en/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -165,6 +165,18 @@ To use it, it is necessary to install and activate it. Refer to the
...
 ## Centreon MBI
 
+### 20.10.3
+
+`June 3, 2021`
+
+#### Bugfixes
+
+- Fixed JQuery issues with MBI web client
+- Fixed job immediate execution that was set in the future
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:175]: Fixed immediate job execution ("immediate" jobs were delayed)

File: fr/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -173,6 +173,19 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
...
+- Correction de l'exécution immédiate des jobs qui était effectuée dans le futur
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:183]: Correction de l'exécution immédiate des jobs (les jobs "immédiats" étaient retardés)

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by Nohzoh — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**